src/sidebar/components/PaginationNavigation.tsx

```diff
@@ -1,24 +1,55 @@
-import { LabeledButton } from '@hypothesis/frontend-shared';
+import {
+  ButtonBase,
+  ArrowLeftIcon,
+  ArrowRightIcon,
+} from '@hypothesis/frontend-shared/lib/next';
+import type { PresentationalProps } from '@hypothesis/frontend-shared/lib/types';
+import type { ButtonCommonProps } from '@hypothesis/frontend-shared/lib/components/input/ButtonBase';
 import classnames from 'classnames';
+import type { JSX } from 'preact';
 
 import { pageNumberOptions } from '../util/pagination';
 
+type NavigationButtonProps = PresentationalProps &
+  ButtonCommonProps &
+  Omit<JSX.HTMLAttributes<HTMLButtonElement>, 'icon' | 'size'>;
+
 /**
- * @typedef PaginationNavigationProps
- * @prop {number} currentPage - The currently-visible page of results. Pages
- *   start at 1 (not 0).
- * @prop {(page: number) => void} onChangePage - Callback for changing page
- * @prop {number} totalPages
+ * Style a navigation button.
+ *
+ * @param {PresentationalProps & ButtonCommonProps & HTMLButtonAttributes} props
  */
+function NavigationButton({ ...buttonProps }: NavigationButtonProps) {
+  return (
+    <ButtonBase
+      classes={classnames(
+        'px-3.5 py-2.5 gap-x-1 font-semibold',
+        // These colors are the same as the "dark" variant of IconButton
+        'text-grey-7 bg-grey-2 enabled:hover:text-grey-9 enabled:hover:bg-grey-3 disabled:text-grey-5 aria-pressed:bg-grey-3 aria-expanded:bg-grey-3'
+      )}
+      {...buttonProps}
+    />
+  );
+}
+
+export type PaginationNavigationProps = {
+  /** 1-indexed page number of currently-visible apge of results */
+  currentPage: number;
```

Comment:
Can you remind me why this uses 1-based numbering, as opposed to 0-based indexing that we normally use?

Reply:
It's been a while and I don't recall, but I'd imagine it was likely because it was skewed toward human-readable UI (e.g. the page numbers are rendered on buttons). We could likely change it.